doc/book/in-depth-guide/first-module.md

```diff
@@ -139,7 +139,7 @@ public function getConfig()
 }
 ```
 
-Reload your application and you'll see that everything remains as it was. Next
+Reload your application and you'll see that everything remains as it was [[This is confusing to me. If it means reload localhost:8080/blog, you get a 404 error as the route hasn't been properly configured yet. If it means reload localhost:8080, obviously everything will remain as it was as you haven't affected the landing page route. Might be my misunderstanding, but spelling out exactly what is meant would help. Also, would be good to keep phrasing consistent, e.g. "refresh your site" vs "reload your application"]]. Next
```

Comment:
Apologies, I think I got confused here with creating the /blog route that comes next (where reloading the page correctly generates a 404 error). That aside, I think it would be clearer to say "reload your application and you'll see that nothing changes". Might seem pedantic, but as the tutorial doesn't specify the previous state of the application my first thought was "how was it? / how should it have been?". 

I know the tutorial assumes you're using the 'album' skeleton app built in the Getting Started tutorial, but I created a new skeleton app from scratch for this tutorial. I think it would be even better to follow it up with your own statement: "creating, registering, and adding empty configuration for a new module has no visible effect on the application." (I have a tech writing background, so I guess I don't think you can ever spell things out too explicitly! : )

Reply:
@obwansan The two changes you propose ("reload your application andyou'll see that nothing changes", and adding the statement about "creating, registering...") make sense - can you update the patch to make those changes, please?

Reply:
Done : )